<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChefsWindow</class>
 <widget class="QMainWindow" name="ChefsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background-color: #7fcdee;
}

QLabel {
	color: #07407b;
}

QSpinBox {
	border: 2.5px solid #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QSpinBox::up-button {
	subcontrol-origin: border;
	subcontrol-position: top right;
	width: 40px;
	height: 18px;
}

QSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	width: 40px;
	height: 18px;
}

QPushButton {
	color: #ffffff;
	font-weight: bold;
	background-color: #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}


QPushButton::hover {
	color:  #f7931e;
	font-weight: bold;
	background-color: #ffffff;
}

#ChefLabel {
	color: #07407b;
	font-weight: bold;
	font-size: 20px;
}

#FoodStockList {
	font-size: 16px;
	background-color: #07407b;
	border: 2.5px solid #f7931e;
	border-radius: 15px;
}

#FoodStockList::item {
	color: #ffffff;
}

#FoodStockList::item::selected {
	color: #f7931e;
}

#SelectedLabel {
	color: #07407b;
	font-weight: bold;
	font-size: 20px;
}

#EditBox {
	background-color: #07407b;
	border: 2.5px solid #f7931e;
	border-radius: 15px;
}

#EditBox QLabel {
	color: #ffffff;
}

#AddLabel {
	font-size: 14px;
	font-style: italic;
}

#RemoveLabel {
	font-size: 14px;
	font-style: italic;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="Top">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ChefLabel">
        <property name="text">
         <string>Chef Interface</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DDSwitchButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Switch to Delivery</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Middle">
      <item>
       <widget class="QListWidget" name="FoodStockList">
        <item>
         <property name="text">
          <string>|     item     |    amount     |     metric     |     date     |</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="flags">
          <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="AddItemButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string>Add Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveItemButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string>Remove Item</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AdminActionsButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string>Admin Actions</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="SelectedLabel">
      <property name="text">
       <string>Currently Selected:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="EditBox" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="AddLabel">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Amount to add:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="AddSpinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddButton">
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="RemoveLabel">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Amount to remove:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="RemoveSpinBox">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveButton">
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
